opensrp-reveal/src/test/java/org/smartregister/reveal/shadow/OfflineManagerShadow.java

```diff
@@ -0,0 +1,24 @@
+package org.smartregister.reveal.shadow;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+import com.mapbox.mapboxsdk.offline.OfflineManager;
+
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+import static org.mockito.Mockito.mock;
+
+/**
+ * Created by Richard Kareko on 1/30/20.
+ */
+
+@Implements(OfflineManager.class)
+public class OfflineManagerShadow {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'OfflineManagerShadow' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)